unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/SceneController.cs

```diff
@@ -228,14 +228,14 @@ public bool ProcessMessage(QueuedSceneMessage_Scene msgObject, out CustomYieldIn
                     case MessagingTypes.ENTITY_CREATE:
                         {
                             if (msgPayload is Protocol.CreateEntity payload)
-                                scene.CreateEntity(payload.entityId);
+                                scene.CreateEntity(payload.entityId.GetHashCode());
 
                             break;
                         }
                     case MessagingTypes.ENTITY_REPARENT:
                         {
                             if (msgPayload is Protocol.SetEntityParent payload)
-                                scene.SetEntityParent(payload.entityId, payload.parentId);
+                                scene.SetEntityParent(payload.entityId.GetHashCode(), payload.parentId.GetHashCode());
```

Comment:
We should NOT use hashcode for converting the string ids to longs. Note that Kernel already uses long for ids internally when passing the messages to Unity. The original string ids are actually base-27 expressions. So the conversion should be made converting from base 27 to base 10. We should untangle this conversion from kernel or re-convert here using this criteria.

Reply:
This is not entirely valid, entity ids happen to be auto increments but that is not the case 100% of times. Hash code seems a better fit for the general case

Reply:
Oh, in a second look it seems like there is a double mapping in the kernel for this

Reply:
Should we then switch the approach GetHashCode for a base36 to base10 conversion? It is efficient enough?